model/flow/transaction.go

```diff
@@ -215,11 +216,20 @@ func (tb *TransactionBody) envelopeCanonicalForm() interface{} {
 		Payload           interface{}
 		PayloadSignatures interface{}
 	}{
-		tb.payloadCanonicalForm(),
+		tb.PayloadCanonicalForm(),
 		signaturesList(tb.PayloadSignatures).canonicalForm(),
 	}
 }
 
+func (tx *Transaction) PayloadMessage() []byte {
+	return fingerprint.Fingerprint(tx.TransactionBody.PayloadCanonicalForm())
+}
+
+// Checksum provides a cryptographic commitment for a chunk content
+func (tx *Transaction) Checksum() Identifier {
```

Comment:
After the latest protocol changes, I believe this should be called `ID`, right @AlexHentschel ?

```suggestion
func (tx *Transaction) ID() Identifier {
```

Reply:
@tim-barry is my current above valid?